app/views/home/index.html.erb

```diff
@@ -1 +1 @@
-<%= link_to "Login", new_user_session_path %>
+<%#= link_to "Login", new_user_session_path %>
```

Comment:
@NickyBobby can you uncomment this? When you merged, it looks like you took your version of application.html.erb which had the navbar and not the version that is on master branch.